src/validations/profile.ts

```diff
@@ -1,10 +1,34 @@
 import { EntityType } from '@dcl/schemas'
 import sharp from 'sharp'
-import { ADR_45_TIMESTAMP } from '.'
+import { ADR_45_TIMESTAMP, ADR_75_TIMESTAMP, validateInRow } from '.'
 import { OK, Validation, validationFailed } from '../types'
+import { parseUrn } from '@dcl/urn-resolver'
 
 /** Validate that given profile deployment includes a face256 thumbnail with valid size */
 const defaultThumbnailSize = 256
+
+export const allowList = new Set([
+  'clap',
+  'dab',
+  'dance',
+  'disco',
+  'dontsee',
+  'fistpump',
+  'hammer',
+  'handsair',
+  'headexplode',
+  'hohoho',
+  'kiss',
+  'money',
+  'raiseHand',
+  'robot',
+  'shrug',
+  'snowfall',
+  'tektonik',
+  'tik',
+  'wave'
+])
```

Comment:
I don't think so. I'd rather be more restrictive and be flexible on demand rather than allow anything by default and then people start hacking this.

Reply:
the rule of thumb we've been following is to not prevent extension of types, e.g. allowExtraProperties:true in schemas. because otherwise a simple art pipeline task (new default emote) would require releasing 12 catalysts and a new version in this library. Also, following the logic of the checks, the list of default values would need to be versioned by timestamp when adding or removing a "base emote"

I'd rather only enforce checks on urn-like emotes and keep the door open for the rest in this case